addons/ingress-nginx/addon.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 Pharos.addon 'ingress-nginx' do
-  version '0.17.1'
+  version '0.20.0'
```

Comment:
I think we should go directly to `0.21.0` (it has security fixes).

Reply:
yeah. need to modify some of the flags for it, I'll update the PR soon for 0.21.0